<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>604</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>967</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="exportWidget" native="true">
     <property name="styleSheet">
      <string>#fileImportHelp {
   color: blue;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Export from:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="exportSnapshotWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="layersList">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_32">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Export to:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="exportTypeWidget">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="usesScrollButtons">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="shapefile_2">
         <attribute name="title">
          <string>Shapefile</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer_18">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="fileBrowserLabel_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string>#fileImportHelp {
   color: blue;
}</string>
            </property>
            <property name="text">
             <string>Filename:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_29">
            <item>
             <widget class="QLineEdit" name="exportFileBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>250</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Enter a file location</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="selectExportFileButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_19">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="spatialite_2">
         <attribute name="title">
          <string>SpatiaLite</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <spacer name="verticalSpacer_23">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_36">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>database:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="spatialiteFileBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="placeholderText">
               <string>database.sqlite</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_37">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>user:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="spatialiteUserBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="placeholderText">
               <string>user</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_24">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="postgis_2">
         <attribute name="title">
          <string>PostGIS</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <spacer name="verticalSpacer_25">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_40">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>host:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportHostBox">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>localhost</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_20">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>port:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportPortBox">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>5432</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_41">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>user:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportUserBox">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>postgres</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_22">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>password:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportPasswordBox">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>postgres</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_42">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>database:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportDatabaseBox">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>schema:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportSchemaBox">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>public</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_43">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string>table:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="pgExportTableBox">
              <property name="styleSheet">
               <string>#fileImportHelp {
   color: blue;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="openLayerCheckbox">
        <property name="text">
         <string>Open in QGIS after exporting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBarExport">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="exportButtonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Close</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>line</zorder>
     <zorder>progressBarExport</zorder>
     <zorder>verticalSpacer_9</zorder>
     <zorder>line_4</zorder>
     <zorder></zorder>
     <zorder>verticalSpacer_3</zorder>
     <zorder>exportTypeWidget</zorder>
     <zorder>openLayerCheckbox</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
